docker/testrunner/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+cd /black-screen
+
+if [[ ! -e /.firstrun ]] && ( [[ ! -d node_modules ]] || [[ -n $FORCE ]] )
```

Comment:
It works correctly, I'm talking about the name. It reads like "is it the first run?" for me. Although, who cares?